docs/static/templates/visualization-template.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Visualization title
+menuTitle: Visualization
+description: Use this template when you write a visualization topic.
+aliases:
+  - /docs/writers-toolkit/latest/templates/concept-template
+weight: 100
+keywords:
+  - keyword 1
+  - keyword 2
+  - keyword 3
+---
+
+<!-- Refer to [Front matter]({{< relref "../../front-matter/" >}}) for more information about how to populate front matter. -->
+
+# Visualization title
+
+<!-- The visualization title is required. This is the name of the visualization as it appears in the UI. For example: Time series.
+
+A visualization topic provides an overview to help end users understand how to best use the visualization type and the options available to them in the user interface (UI). Visualization topics always include conceptual and reference content and may include task content.
+
+Refer to the [Visualization topic documentation](https://grafana.com/docs/writers-toolkit/writing-guide/topic-types/visualization/) for guidelines on writing a visualization topic.
+-->
+
+Introduce the concept.
+
+<!-- The introduction is required. Add an introduction to the visualization that explains what visualization type can help the user do. -->
+
+## Supported data formats
+
+<!-- Provide guidance about which data formats are supported by the visualization type with example use cases. Does not apply to a widget topic. -->
+
+## [A] options
+
+<!-- For each drop-down section of options in the edit panel add a section in the topic with the name of the UI section, followed by the word "options." This doesn't include the sections linked from the **Other options** section. -->
+
+## [B] options
+
+<!-- For each drop-down section of options in the edit panel add a section in the topic with the name of the UI section, followed by the word "options." This doesn't include the sections linked from the **Other options** section. -->
+
+## Other options
```

Comment:
Note to DataViz reviewers, the content of this section of the template may change as we finish our review of common visualization options and how they're implemented for each visualization type. However, that shouldn't be a blocker to getting an initial template created.